style.css

```diff
@@ -377,6 +518,27 @@ button {
   font-family: 'Poppins', sans-serif;
 }
 
+.button4 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-weight: 500;
+  box-sizing: border-box;
+  line-height: 24px;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  border: 1px solid #6070ff;
+  cursor: pointer;
+  color: #6070ff;
+  background-color: #fff;
+  gap: 10px;
+  padding: 12px;
+  width: 149px;
+  margin-left: 40px;
+  border-radius: 8px;
+  letter-spacing: 0.03em;
+}
+
 .about_section_container {
   display: flex;
   flex-direction: column;
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly try reducing the height to reduce the too much space to match the Figma design below:

*figma*
![small-space](https://user-images.githubusercontent.com/59397159/172935202-19a7762d-13f4-4095-8243-43c487321d6b.png)

*your design*
![big-space](https://user-images.githubusercontent.com/59397159/172935208-a8b87d7d-7931-4239-937c-9a3d6a7e324c.png)